gui.py

```diff
@@ -352,6 +353,14 @@ def update_status_tracking(self, status, text_color: str = 'white', background_c
         Logger.getLogger().info('update_status_tracking in gui')
         self.win.FindElement('status-tracking').Update(status, text_color=text_color, background_color=background_color)
 
+    # STATUS SLEWING
```

Comment:
I think these kinds of comments are redundant. We are already using python Docstring inside the methods, so there is no need for these.